src/Provider/Doctrine/Auditing/Logger/Middleware/DHConnection.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DH\Auditor\Provider\Doctrine\Auditing\Logger\Middleware;
+
+use Doctrine\DBAL\Driver\Connection as ConnectionInterface;
+use Doctrine\DBAL\Driver\Result;
+use Doctrine\DBAL\Driver\Statement;
+use Doctrine\DBAL\ParameterType;
+
+/**
+ * @interal
+ */
+final class DHConnection implements ConnectionInterface
```

Comment:
Impossible to put flushers to DHConnection, because I can't get it in DoctrineSubscriber - `getWrappedConnection` is also deprecated.
New method`getNativeConnection` will return not `DHConnection`, but directly Postgres/Mysql/other connection